xst.c

```diff
@@ -88,6 +88,7 @@ xrdb_load(void)
 		XRESOURCE_LOAD_INTEGER("borderpx", borderpx);
 		XRESOURCE_LOAD_INTEGER("borderless", borderless);
 		XRESOURCE_LOAD_INTEGER("cursorshape", cursorshape);
+		XRESOURCE_LOAD_INTEGER("disableAlphaCorrection", disableAlphaCorrection);
```

Comment:
you forgot to add `alpha` here

Reply:
I'm thinking we can leave it as `opacity` at the xresources level